neqo-crypto/build.rs

```diff
@@ -147,10 +148,9 @@ fn build_nss(dir: PathBuf) {
     let mut build_nss = vec![
         String::from("./build.sh"),
         String::from("-Ddisable_tests=1"),
+        String::from("--static"),
```

Comment:
I don't understand this change.  Can you explain?

Reply:
I had queued up a change with a comment that I forgot to push. 
```
// Generate static libraries in addition to shared libraries.
```
Basically, `--static` seems to not only build static libraries, it seems to build static libraries **in addition** to dynamic ones. So no reason to not always use it.